### PR TITLE
Insertion sort

### PR DESCRIPTION
**Pull request by wosunkwo**

### CODE REVIEW

**Comment by wosunkwo** [comment]
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

File: code401-challenges/src/test/java/code401/challenges/insertionSort/InsertionSortTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        assertArrayEquals("This should return true if the expected output and the array the method returns are the same", expectedOutout, classUnderTest.InsertionSort(edgeArr));
+    }
+
+    @Test
+    public void expectedFailure(){
+        int[] expectedOutout = new int[] {};
+        assertArrayEquals("This should return true if the expected output and the array the method returns are the same", expectedOutout, classUnderTest.InsertionSort(expectedFailure));
+    }
+
+    }
```
Comment (evanslaton) [inline code401-challenges/src/test/java/code401/challenges/insertionSort/InsertionSortTest.java:39]: Improperly indented!
